src/jvmMain/kotlin/ru/spbu/retest/view/ProgressBar.kt

```diff
@@ -0,0 +1,39 @@
+package ru.spbu.retest.view
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.dp
+import ru.spbu.retest.ViewModel
+import kotlin.time.ExperimentalTime
+
+
+@Composable
```

Comment:
```suggestion
import kotlin.time.ExperimentalTime

@Composable
```
Две пустые строки подряд вроде как по стайлгайду никогда не ставятся